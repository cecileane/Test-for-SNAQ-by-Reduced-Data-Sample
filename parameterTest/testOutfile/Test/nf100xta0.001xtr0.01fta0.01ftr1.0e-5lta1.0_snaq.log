optimization of topology, BL and inheritance probabilities using:
 hmax = 1,
 tolerance parameters: ftolRel=1.0e-5, ftolAbs=0.01,
                       xtolAbs=0.001, xtolRel=0.01.
 max number of failed proposals = 100, liktolAbs = 1.0.
rootname for files: nf100xta0.001xtr0.01fta0.01ftr1.0e-5lta1.0_snaq
BEGIN: 2 runs on starting tree (2,3,(4,(5,(1,6):0.386):1.19):0.028);

main seed 71466
seed: 71466 for run 1
Sat Oct  1 17:35:13 2016

 BEGIN SNaQ for run 1, seed 71466 and hmax 1 changed starting topology by NNI move

Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 114 (failures: 100) with -loglik=128.22643 and ht_min=[0.30771,0.66107,0.4599,1.7391,8.87903]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 2		 1		 1	 |	 NA 		 NA 		 NA 
mvorigin	 25		 25		 0	 |	 0		 0		 --
mvtarget	 25		 25		 0	 |	 0		 0		 --
chdir		 4		 4		 0	 |	 0		 0		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 58		 47		 0	 |	 4		 4		 --
Total		 114		 102		 1	 |	 4		 4		 4
Proportion	 -- 		 1.0		 0.0	 |	 -- 		 1.0		 1.0

 FINISHED SNaQ for run 1, -loglik of best 128.2264279465274
(2,(((5,(6)#H7:::0.6922866281561859):1.7391023766243179,3):0.4598963237323319,4):0.6610726742513429,(1,#H7:::0.30771337184381403):8.879029416944668);
---------------------
seed: 55767 for run 2
Sat Oct  1 17:35:19 2016

 BEGIN SNaQ for run 2, seed 55767 and hmax 1
Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 146 (failures: 100) with -loglik=128.22643 and ht_min=[0.30771,0.66107,0.4599,1.73908,8.87903]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 2		 2		 1	 |	 NA 		 NA 		 NA 
mvorigin	 36		 36		 0	 |	 0		 0		 --
mvtarget	 32		 32		 0	 |	 0		 0		 --
chdir		 8		 8		 0	 |	 0		 0		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 68		 57		 2	 |	 4		 4		 --
Total		 146		 135		 3	 |	 4		 4		 4
Proportion	 -- 		 1.0		 0.0	 |	 -- 		 1.0		 1.0

 FINISHED SNaQ for run 2, -loglik of best 128.22642794630892
((4,((5,(6)#H7:::0.6922875520818124):1.7390821197985344,3):0.4598963242029504):0.6610724058317815,(1,#H7:::0.3077124479181876):8.879031176014387,2);
---------------------

Sat Oct  1 17:35:28 2016
MaxNet is ((4,((5,(6)#H7:::0.6922875520818124):1.7390821197985344,3):0.4598963242029504):0.6610724058317815,(1,#H7:::0.3077124479181876):8.879031176014387,2); 
with -loglik 128.22642794630892
