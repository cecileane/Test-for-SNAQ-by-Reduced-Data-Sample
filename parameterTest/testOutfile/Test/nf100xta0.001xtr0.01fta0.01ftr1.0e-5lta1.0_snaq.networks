((4,((5,(6)#H7:::0.6922875520818124):1.7390821197985344,3):0.4598963242029504):0.6610724058317815,(1,#H7:::0.3077124479181876):8.879031176014387,2);, with -loglik 128.22642794630892 (best network found, remaining sorted by log-pseudolik; the smaller, the better)
((4,((5,(6,#H7:::0.28644129714254235):4.847109923400502):0.5202460769413303,3):0.49497090564090407):1.5257877885735878,(1)#H7:::0.7135587028574577,2);, with -loglik 126.81066130373114
(#H7:::0.380525898271447,(1,(6,(((4)#H7:::0.619474101728553,3):0.025290519115773704,5):0.21328904371720195):1.173894608955977):9.662351432240358,2);, with -loglik 171.91758643955745
(#H7:::0.44249542437983536,4,((5,(6,(1,(2)#H7:::0.5575045756201646):3.699085118623487):0.14354589471453666):0.03934601886904581,3):1.183262760914725);, with -loglik 196.6450792628182
((4,(3)#H7:::0.554954890528467):0.10517666027095207,(1,(6,(#H7:::0.445045109471533,5):1.614738567929738):1.019479064021437):0.10416726349321094,2);, with -loglik 214.35629109754146
((4,((5)#H7:::0.5093418150298099,3):1.1595828215578436):0.19817544320065258,(1,(6,#H7:::0.49065818497019015):5.931711925800473):0.04296652658713674,2);, with -loglik 231.1286095853302
