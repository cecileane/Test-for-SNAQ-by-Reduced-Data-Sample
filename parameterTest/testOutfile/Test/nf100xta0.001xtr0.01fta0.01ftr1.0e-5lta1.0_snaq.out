((4,((5,(6)#H7:::0.6922875520818124):1.7390821197985344,3):0.4598963242029504):0.6610724058317815,(1,#H7:::0.3077124479181876):8.879031176014387,2); -Ploglik = 128.22642794630892
 Dendroscope: ((4,((5,(6)#H7):1.7390821197985344,3):0.4598963242029504):0.6610724058317815,(1,#H7):8.879031176014387,2);
 Elapsed time: 14.519868936 seconds in 2 successful runs
-------
List of estimated networks for all runs (sorted by log-pseudolik; the smaller, the better):
 ((4,((5,(6)#H7:::0.6922875520818124):1.7390821197985344,3):0.4598963242029504):0.6610724058317815,(1,#H7:::0.3077124479181876):8.879031176014387,2);, with -loglik 128.22642794630892
 (2,(((5,(6)#H7:::0.6922866281561859):1.7391023766243179,3):0.4598963237323319,4):0.6610726742513429,(1,#H7:::0.30771337184381403):8.879029416944668);, with -loglik 128.2264279465274
-------
